configure/options.md

```diff
@@ -0,0 +1,39 @@
+Changing MySQL Options
+============================================================================
+
+MySQL allows the option to configure the database with a configuration file. You can pass the MySQL configuration options to the cluster in the following ways:
+* CR.yaml
+* ConfigMap
+
+### Edit the CR.yaml
+
+Edit the configuration section of the deploy/cr.yaml. See the [PXC section]( https://percona.github.io/percona-xtradb-cluster-operator/configure/operator).
```

Comment:
It would be good to have an example here, for example an uncommented fragment of cr.yaml starting from this line: https://github.com/percona/percona-xtradb-cluster-operator/blob/3d92c5c3a0b05fcb22a4b19c22c982071415b10e/deploy/cr.yaml#L15